src/cinder/gl/Fbo.cpp

```diff
@@ -305,7 +305,7 @@ void Fbo::initMultisamplingSettings( bool *useMsaa, bool *useCsaa, Format *forma
 }
 
 // Iterate the Format's requested attachments and create any we don't already have attachments for
-void Fbo::prepareAttachments( const Fbo::Format &format, bool multisampling )
+void Fbo::prepareAttachments( const Fbo::Format &format, bool /*multisampling*/ )
```

Comment:
If `multisampling` isn't used, shouldn't it be removed as an argument altogether? This is a non-virtual, protected method.